pages/deployment/[uuid].tsx

```diff
@@ -163,6 +155,10 @@ export default function Deployment() {
     setAnchorEl(null)
   }
 
+  const onSelectedTagChange = (event: SelectChangeEvent) => {
+    setSelectedImageTag(event.target.value as string)
```

Comment:
The type assertion here isn't needed as `event.target.value` is already considered a `string`